frontend/public/index.html

```diff
@@ -15,6 +15,8 @@
     <meta property="og:type" content="website"/>
     <meta property="og:description" content="100% free and interactive Python course for beginners"/>
     <meta property="og:image" content="https://futurecoder.io/static/logo/bordered.png"/>
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```

Comment:
Adding the manifest.json is the main thing that made the Lighthouse report say it's an installable PWA. But as mentioned in the PR description, it's not actually working yet.

Reply:
![image](https://user-images.githubusercontent.com/2420688/155920980-797f359c-7580-4cfc-9cd7-b7e57f22f247.png)

Reply:
It "works" for a while, but then you see some `ERR_INTERNET_DISCONNECTED` errors in the logs, and after that point, upon refresh, it's just:

![image](https://user-images.githubusercontent.com/2420688/155921305-a9ff8eb0-2b4f-4643-b228-f7b2efc4ad78.png)

so, the caching or precaching seems to be messed up in the service worker ... But since I've followed the official CRA instructions and example, I'm not really sure why.

Reply:
Hm, at least as of [0912be0](https://github.com/alexmojaki/futurecoder/pull/313/commits/0912be0b8e26fa6c325824e3b9134f322d6a8d6a) (just pushed), it seems like it "works" until going offline. The bad part is that even when back online, it won't work again until all tabs have been closed.

Reply:
@alexmojaki Do you have any ideas based on the URLs in the error message?

Reply:
Those URLs are the first 4 things I see loading in my network tab in normal development. They're just the usual assets. So it sounds like nothing is loading at all. But it also looks like you're still using the dev server instead of building, maybe that messes things up?

Reply:
Sounds plausible that it's from something to do with using the dev server. Things weren't working for me when I built and served it though, on this branch... but it could have been some local mess.